cluster/images/kubemark/Dockerfile

```diff
@@ -14,6 +14,6 @@
 
 # The line below points to debian:jessie as of 2019-10-23. The SHA should be
 # kept in sycn with debian_jessie definition in the WORKSPACE file.
-FROM debian@sha256:e25703ee6ab5b2fac31510323d959cdae31eebdf48e88891c549e55b25ad7e94
+FROM gcr.io/distroless/base:latest
```

Comment:
Please use the same sha as in WORKSPACE:

aka `gcr.io/distroless/base@sha256:7fa7445dfbebae4f4b7ab0e6ef99276e96075ae42584af6286ba080750d6dfe5`